src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -298,6 +298,7 @@
             monitoring - turn on Log Analytics monitoring. Uses the Log Analytics Default Workspace if it exists, else creates one.
                          Specify "--workspace-resource-id" to use an existing workspace.
                          If monitoring addon is enabled --no-wait argument will have no effect
+            azure-policy - enable Azure policy.
```

Comment:
Nit (optional) would be nice to have a one liner about policy. Something like "enable Azure policy to provide fine grain control of X"

Reply:
We could say:

"The Azure Policy add-on for AKS enables at-scale enforcements and safeguards on your clusters in a centralized, consistent manner. Learn more at aka.ms/aks/policy."

Reply:
sounds good, will add it.